supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -5,5 +5,122 @@
 [[release-notes]]
 == Release notes
 
+[[release-notes-doc-texts]]
+=== Conventions for phrasing Release Notes doc texts
+
+A doc text is a short description of an engineering Bugzilla or Jira issue that is published in product Errata Advisories and Release Notes. Engineering supplies draft content, which is a summary of the issue, then technical writers edit or rewrite the draft content in accordance with the IBM Style Guide. You can write doc texts in more than one way, depending on the issue type: 
```

Comment:
Should we cite this reference as _The IBM Style Guide_?

Reply:
Done, thanks.